content/governance.md

```diff
@@ -145,9 +147,37 @@ A vote on a single proposal is considered successful if at least two thirds of t
 
 If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
 
+## On- / Offboarding
+
+The On- / Offboarding section is informational and can be changed by [lazy consensus](#consensus) unless challenged.  If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### Onboarding
+
+The new member is
+* added to the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR.
+* announced on the [developers mailing list][devs] by an existing team member. Ideally, the new member replies in this thread, acknowledging team membership.
+* added to the [GitHub organization][gh] as _Owner_.
+* optionally added to the community, junkyard, and related repositories.
+* added to the [team mailing list][team].
+* announced to CNCF.
+* given access to the shared password storage.
+
+### Offboarding
+
+The ex-member is
+* removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
+* removed from the [GitHub organization][gh] and related organizations and repositories.
+* removed from the [team mailing list][team].
+* announced as removed to CNCF. We will explicitly request reconfirmation of removal from CNCF.
+* removed from shared password storage. All passwords, API tokens, etc are cycled.
+* not allowed to call themselves an active team member any more, nor allowed to imply this to be the case.
+* added to a list of alumnae if they agree to it.
```

Comment:
This is now only the feminine form. Personally, I'm fine with it, definitely better than using the masculine form in an anyway all-male club. But my original suggestion was including both genders, which is probably the best in general.

Reply:
See above; "ex-members" ok? I don't like gendered terms at all for brevity and as they exclude non-binary people.

Reply:
Beorn approved PR in full.